scripts/collect_release_evidence.py

```
@@ -0,0 +1,666 @@
+    elif validator.get("passed") is not True or validator.get("exit_code") != 0:
+        errors.append(f"{field}.validator must have passed with exit_code 0")
+    if result_path and result_path.exists() and repo_root is not None:
+        validator_path = repo_root / "benchmarks" / "validate_benchmark_result.py"
+        completed = subprocess.run(
+            [sys.executable, str(validator_path), str(result_path)],
+            cwd=repo_root,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            check=False,
+        )
```

Ruff (0.15.10)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)